generate/src/contract.rs

```diff
@@ -87,6 +89,17 @@ impl Context {
         };
         let contract_name = util::ident(raw_contract_name);
 
+        let mut method_aliases = HashMap::new();
+        for (signature, alias) in args.method_aliases.into_iter() {
+            let alias = syn::parse_str(&alias)?;
+            if method_aliases.insert(signature.clone(), alias).is_some() {
+                return Err(anyhow!(
```

Comment:
Here you are not checking for duplicate alias?

Reply:
No because we will get compile errors on duplicate aliases as there will be two methods with the same identifier.

Reply:
I'll add a note.